test/views/html/ViewSpec.scala

```diff
@@ -16,14 +16,30 @@
 
 package views.html
 
+import config.ConfigDecorator
+import models.PertaxContext
 import org.jsoup.Jsoup
 import org.jsoup.nodes.Document
+import org.scalatest.Assertion
+import play.api.i18n.{Lang, Messages}
+import play.api.i18n.MessagesApi
+import play.api.test.FakeRequest
+import util.BaseSpec
 
-trait ViewSpec {
+trait ViewSpec extends BaseSpec {
 
```

Comment:
If a ViewSpec is really a BaseSpec then there are a few other places to tidy up where Specs are still extending both ViewSpec with BaseSpec